src/integration/pages/storyPage/crossPlatformTests.js

```diff
@@ -12,8 +12,65 @@ export default () => {
     expect(imageEl).toBeInTheDocument();
     expect(imageCaptionEl).toBeInTheDocument();
     expect(imageCaptionEl.textContent).toBeTruthy();
-    expect(imageCaptionEl.textContent).toMatchInlineSnapshot(
-      `"Pie de foto, Llegó el día de la salida de Reino Unido de la UE."`,
-    );
+    expect(imageCaptionEl.textContent).toMatchSnapshot();
+  });
+
+  describe('Social Embeds', () => {
+    it('I can see the skip link', () => {
+      const skipLinkEl = document.querySelector(
+        'a[href="#skip-youtube-content-1"]',
+      );
+
+      expect(skipLinkEl).toBeInTheDocument();
+      expect(skipLinkEl.textContent).toBeTruthy();
+      expect(skipLinkEl.textContent).toMatchSnapshot();
+    });
+
+    it('I can see the skip link destination', () => {
+      const skipLinkDestinationEl = document.getElementById(
+        'skip-youtube-content-1',
+      );
+
+      expect(skipLinkDestinationEl).toBeInTheDocument();
+      expect(skipLinkDestinationEl.textContent).toBeTruthy();
+      expect(skipLinkDestinationEl.textContent).toMatchSnapshot();
+    });
+
+    /**
+     * Rich embeds.
+     */
+    const hasRichYouTubeEmbed = [
+      document.querySelector('iframe[src*="https://www.youtube.com/embed/"]'),
+      document.querySelector('amp-youtube'),
+    ].find(el => !!el);
+
+    if (hasRichYouTubeEmbed) {
+      it('I can see the caption accompanying a YouTube embed', () => {
+        const figCaptionEl = document.querySelector(
+          'a[href="#skip-youtube-content-1"] + figure > figcaption',
+        );
+
+        expect(figCaptionEl).toBeInTheDocument();
+        expect(figCaptionEl.textContent).toBeTruthy();
+        expect(figCaptionEl.textContent).toMatchSnapshot();
+      });
+    }
+
+    /**
+     * Fallbacks.
+     */
+    const hasFallback = !!document.getElementById('skip-facebook-content-1');
+
+    if (hasFallback) {
+      it('I can see a link to the original content in a fallback', () => {
```

Comment:
Is it worth specifying Facebook in the test description?

Reply:
This test is not specific to Facebook and would look the same for LinkedIn or any other provider we don't support – we just want to ensure "[We] can see a link to the original content in a fallback".

N.B. We know that Facebook doesn't render as a 'rich embed', so that's why it can be seen here, but if I personalise this test by mentioning Facebook in its title, I figured that might risk it being replicated unnecessarily for other providers.

Reply:
I am happy to leave it as is for now, given that these tests are specifically for that URL. May need to revisit some of the query selectors once we add more STYs for other services.